library/ssl_cli.c

```diff
@@ -891,6 +891,8 @@ static int ssl_write_client_hello( mbedtls_ssl_context *ssl )
         *p++ = (unsigned char)( ciphersuites[i]      );
     }
 
+	MBEDTLS_SSL_DEBUG_MSG(3, ("client hello, got %d ciphersuites", n));
+
```

Comment:
Perhaps we should add a log message  when adding `MBEDTLS_SSL_EMPTY_RENEGOTIATION_INFO`?